### ISSUE TITLE
AMOEBA dipole profile calculation

### ISSUE THREAD

**Issue by rowanwalker96** [Inniag/openmm-scripts-amoeba#2]
Hello,

Thanks for sharing your AMOEBA OpenMM scripts! I have a question regarding how you are calculating your dipole moment profile - it seems that you are using the center of geometry of the water molecules to calculate the monopole charge weighted positions from, is this correct? Then you are binning the vector sum of the monopole charge weighted positions, lab frame permanent dipoles and induced dipoles on the center of geometry?

Unfortunately I cannot seem to reproduce the z component of the dipole moment of the AMOEBA14 water model if I use the center of geometry to calculate my monopole charge weighted positions term...

I asked a similar question on the OpenMM github page (https://github.com/openmm/openmm/issues/3141), and it was suggested that the correct reference point to use instead would be the center of mass.

I know the dipole moment profiles probably don't change all that much but I am finding that it does become important for example in studying the z component of the dipole moment of nanoconfined water molecules right at an interface.

Thanks for reading!

**Comment by Inniag** [comment]
Hi, glad to hear my code is helpful to other people's research.

Regarding your question:
- Yes, the calculation of the total molecular dipole moment is done by vector summing the induced and permanent dipole moments with the position weighted sum of the monopole charges. This is tantamount to taking the center of geometry of the molecule as reference point.
- The center of mass is only used as the position of the molecule when determining the position of the molecule along the center line of the ion channel pore. I don't see how the center of mass would factor in to the calculation of dipole moments though. Dimensional analysis would suggest mass weighting plays no role in calculating dipole moments from charge distributions.
- You could of course use a different reference point for the dipole calculation. By my understanding of the theory, the reference point can be chosen according to convention and I have no idea, which convention the makers of the AMOEBA force field followed. I also think I recall a theoretical result that the reference point does not matter so long as the overall system is charge neutral. Can't remember the reference right now, but it's probably this: https://oxford.universitypressscholarship.com/view/10.1093/acprof:oso/9780199672394.001.0001/acprof-9780199672394-chapter-3
- I don't understand how any of this could possibly impact the z-component of the dipole moment, given that water is a planar molecule. If I am not totally mistaken, the dipole moment due to monopole charges will always lie in the plane spanned by the three molecules and as both the center of geometry and center of mass lie in this plane as well, the z-component of the net dipole moment should be independent of this? Or am I overlooking something obvious here.

Hope this helps.

**Comment by Inniag** [comment]
OK, scratch that last comment, I had forgotten that AMOEBA defines the z-axis as the bisector of the water molecules - it's been two years since I last worked on this.

But then the answer is obvious, if you want to reproduce the exact numerical value from the AMOEBA14 paper, you will have to use their convention for specifying reference points. If center of oxygen is the right reference point for you, depends on your use case. And on the experimental setup you want to compare to. Just make sure you are consistent.

**Comment by rowanwalker96** [comment]
Thanks so much for getting back to me! Sorry yes I agree the dipole moment does not change with respect to the reference point used to calculate it, my question then is just about where I should plot the location of the resulting dipole vector itself in order to create a profile of the dipole moment density. I have been using the center of mass (the position of the water molecule). 

My profiles of the z component of the dipole moment are only different because I have a sharp capacitor plate at which some molecular reference points are able to get closer to than others.

**Comment by Inniag** [comment]
For the location of the dipole moment, I also used the center of mass, which as at least consistent with the notion of position in the dynamical sense.

However, I only looked at water molecules and for those, the difference between center of mass, center of geometry, and position of oxygen atom is only minuscule. We are talking about a difference of well under an angstrom here, which is small compared to the length scale of any remarkable features of the dipole moment profile along the length of a transmembrane protein. Also note that the position along the protein pore that I considered is already a simplification, because the amino acid side chains are flexible and in motion. This effect results in a statistical uncertainty that is much greater than the difference between the different definitions of dipole moment location.

Of course, your situation may be different, if you look at larger molecules.
